.moban.yaml

```diff
@@ -25,16 +26,23 @@ packages:
 configuration:
   template_dir:
     - .moban.dt/
-    - ../coala-mobans/templates/
-    - ../coala-mobans/assets/
-    - ../setupmobans/templates/
+    - ../coala-mobans/
```

Comment:
please use the same as other repos

```
    - ../coala-mobans/templates/
    - ../coala-mobans/assets/
```

We need consistency so that we can meta-program an automatic sync of the dependent repos.